pages/educational/index.tsx

```diff
@@ -0,0 +1,168 @@
+      width: 124,
+      message: (
+        <>
+          <h4 className="mb-2 mt-5 text-sm font-bold">{name}</h4>
+          <PComponent children={description} />
+        </>
+      ),
+    });
+    return <HorizontalImageLayout images={solutions.map(solutionPropsMap)} />;
+  };
+
+  const DownloadWhitepaperLink = ({ children }) => (
+    <Link href={node.whitepaperFile} passHref legacyBehavior>
+      <a target="_blank">{children}</a>
+    </Link>
+  );
+  const educationalRenderer: Components<{
+    Solutions: {
+      educationalSolutions: {
+        solutionImage: string;
+        name: string;
+        description: string;
+      }[];
+    };
+    VideoEmbed: {
+      url: string;
+    };
+    Whitepaper: {
+      title: string;
+      description: string;
+      buttonText: string;
+    };
+    BookingForm: Record<string, never>;
+    ContactUs: {
+      buttonText: string;
+      link: string;
+    };
+  }> = {
+    ...componentRenderer,
+    h3: ({ children }) => (
+      <h3 className="mb-3 mt-10 text-sswRed">{children}</h3>
+    ),
+    p: PComponent,
+    Solutions: ({ educationalSolutions }) => (
+      <SolutionElements solutions={educationalSolutions} />
+    ),
+    VideoEmbed: ({ url }) => (
+      <div className="relative h-0 overflow-hidden pb-9/16">
+        <div className="absolute h-full w-full">
+          <ReactPlayer url={url} width="100%" height="100%" controls={true} />
+        </div>
+      </div>
+    ),
+    Whitepaper: ({ title, description, buttonText }) => (
+      <div className="mt-4 border-1 border-gray-300 bg-gray-125 px-6 py-4">
+        <h4 className="mb-3 mt-6">{title}</h4>
+        <p className="mb-4">{description}</p>
+        <div className="flex justify-center">
+          <DownloadWhitepaperLink>
+            <Button ripple className="done mx-2 !h-10 pl-3">
+              {buttonText}
+            </Button>
+          </DownloadWhitepaperLink>
+        </div>
+      </div>
+    ),
+    BookingForm: () => (
+      <BookingForm recaptchaKey={props.env["GOOGLE_RECAPTCHA_KEY"]} />
+    ),
+    ContactUs: ({ buttonText, link }) => (
+      <div className="mb-16 flex justify-center">
+        <a href={link}>
+          <Button
+            ripple
+            className="!h-10 !bg-gray-900 bg-arrow-right bg-right bg-no-repeat pl-3 pr-8 text-sm"
+          >
+            {buttonText}
+          </Button>
+        </a>
+      </div>
+    ),
+  };
+
+  return (
+    <Layout>
+      <SEO seo={{ ...node.seo, canonical: "/educational" }} />
+      <Container className="flex-1" size="custom">
+        <DownloadWhitepaperLink>
+          <Image
+            src={node.bannerImg}
+            width={1312}
+            height={0}
+            alt="SSW Educational Banner"
+            sizes="100vw"
+          />
+        </DownloadWhitepaperLink>
+        <Breadcrumbs
+          path={"educational"}
+          suffix=""
+          title={"SSW Consulting Educational - Sydney, Brisbane, Melbourne"}
+        />
```

Comment:
These hardcoded strings are not ideal.

Copy the approach used by the consulting page template

https://github.com/SSWConsulting/SSW.Website-v3/blob/0cc5673ad1ca0ac88daba4ce2bea45b882184082/pages/consulting/[filename].tsx#L62